buyer-api/src/operations/registerPayment.js

```diff
@@ -63,5 +68,7 @@ export const registerPayment = async (notarizationRequestId, pauseQueue) => {
 
   await addTransactionJob('RegisterPayment', payload);
 
+  await currentPaymentsAmount.update('current_amount', currentAmount => toBN(currentAmount).add(amount), 0);
```

Comment:
If we have two pending payments, and just one of them finalizes, it will set the amount to zero and we will miss the other/s. I think we need to keep a separate record for each payment and retrieve the sum when query, and delete on confirmation

Reply:
Nope, that 0 is the value that is set if there's no record of `current_amount` (i.e.: the first time you register a payment after this change)

Reply:
🤦‍♂You're right!